internal/pkg/cli/app_init.go

```diff
@@ -256,6 +273,31 @@ func (o *initAppOpts) askDockerfile() error {
 	return nil
 }
 
+func (o *initAppOpts) askAppPort() error {
+	if o.AppPort != 0 {
+		return nil
+	}
+
+	port, err := o.prompt.Get(
+		fmt.Sprintf(fmtAppInitAppPortPrompt),
+		fmt.Sprintf(fmtAppInitAppPortHelpPrompt),
+		validateApplicationPort,
+		prompt.WithDefaultInput("80"),
+	)
+	if err != nil {
+		return fmt.Errorf("failed to get port: %w", err)
+	}
+
+	portUint, err := strconv.ParseUint(port, 10, 16)
+	if err != nil {
+		return fmt.Errorf("failed to parse port string: %w", err)
+	}
+
+	o.AppPort = uint16(portUint)
```

Comment:
Should we do the same validation as what we did to the flag as well? Like errors out if `-lt 1 or -gt 65535`?

Reply:
There's validation in the Get() call on line 281 which ensures that the input is in the uint16 range (that's where string validation happens, since Get returns a string). Then we convert the input string to uint16.

Reply:
![image](https://user-images.githubusercontent.com/25392995/76128061-26ee6400-5fb8-11ea-99bb-7d53d730acab.png)

Reply:
oh I missed that one. Then it make senses to me!